compose-builder/add-security.yml

```diff
@@ -216,6 +216,7 @@ services:
       KONGURL_SERVER: kong
       SECRETSTORE_HOST: edgex-vault
       SECRETSTORE_TOKENPATH: /tmp/edgex/secrets/edgex-security-proxy-setup/secrets-token.json
+      EDGEX_SECURITY_SECRET_STORE: "true"
```

Comment:
add `common-security.env` to `env_file` section instead and remove `SECRETSTORE_HOST: edgex-vault` which is in  `common-security.env`

Reply:
not a fan to remove `SECRETSTORE_HOST: edgex-vault` at this point since that one have more wide-spread effect and not sure it will work. I'll add common-security-.env.

Reply:
added `common-security.env`

Reply:
`SECRETSTORE_HOST=edgex-vault` is in `common-security.env`, so not need again here.
Please remove as it isn't on any of the other services that use  `common-security.env`.

Reply:
it needs to be there for a couple of `security-bootstrapper`'s entrypoint script

Reply:
consul and kong-db, specifically

Reply:
Right, and both of those get `SECRETSTORE_HOST` from `common-security.env`.

What I an saying is we don't need `SECRETSTORE_HOST: edgex-vault` in `environment` section for  `proxy-setup`' since `common-security.env`  sets `SECRETSTORE_HOST`.

Reply:
got it, i missed that part, thanks!